### PR TITLE
Added a new meme

### PR DESCRIPTION
**Pull request by nxtCP**

### CODE REVIEW

**Review by OpenSourceSimon — APPROVED** [review]

**Comment by LevaniVashadze** [comment]
Dont merge into master, we merge into the development branch and the function name of the command is the same as before and the slash command name has spaces wich will not work most probably if you have tested it

File: cogs/Memes.py [hunk 1]
```diff
@@ -18,6 +18,12 @@ async def onlyfans(self, inter):
         await inter.send(
             "https://media.wired.com/photos/59548ac98e8cc150fa8ec379/master/w_2560%2Cc_limit/GettyImages-56196238.jpg"
         )
+    @commands.slash_command(name="boners with lewis", description="Boners with lewis")
+    async def onlyfans(self, inter):
```
Comment (LevaniVashadze) [inline cogs/Memes.py:22]: We already have an onlyfand named function above wich you copied

File: cogs/Memes.py [hunk 2]
```diff
@@ -18,6 +18,12 @@ async def onlyfans(self, inter):
         await inter.send(
             "https://media.wired.com/photos/59548ac98e8cc150fa8ec379/master/w_2560%2Cc_limit/GettyImages-56196238.jpg"
         )
+    @commands.slash_command(name="boners with lewis", description="Boners with lewis")
```
Comment (LevaniVashadze) [inline cogs/Memes.py:21]: Spaces won't work in the name

**Review by LevaniVashadze — CHANGES_REQUESTED** [review]
See comments and merge into dev

File: cogs/Memes.py (hunk 2 above)
Comment (OpenSourceSimon) [reply cogs/Memes.py:21]: I think it does work, because the tag commands have a space in them

Comment (LevaniVashadze) [reply cogs/Memes.py:21]: Those are subcommands

Comment (OpenSourceSimon) [reply cogs/Memes.py:21]: Oh okay didn't know that

**Pull request closed without merging**